Sources/SwiftDocC/Semantics/Metadata/TitleHeading.swift

```diff
@@ -0,0 +1,41 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+/// A directive for customizing the text of a page's title heading.
+/// 
+/// The ``headingText`` property will override the page's default title heading.
+///
+/// @TitleHeading accepts an unnamed parameter containing containing the page's title heading.
+/// 
+/// This directive is only valid within a top-level ``Metadata`` directive:
+/// ```markdown
+/// @Metadata {
+///    @TitleHeading("Release Notes")
+/// }
+/// ```
+public final class TitleHeading: Semantic, AutomaticDirectiveConvertible {
+    public let originalMarkup: BlockDirective
+
+    /// An unnamed parameter containing containing the page-title’s heading text.
+    @DirectiveArgumentWrapped(name: .unnamed)
+    public var headingText: String
```

Comment:
Sorry – just seeing this now. The name of this property will appear in documentation so I think we should make sure it aligns with the other directives.

In this case, I think the `Text` suffix breaks with the existing style we have. Maybe just `heading` would work?

 ```suggestion
    public var heading: String
```

That would match with `DisplayName` which use `name` as its unnamed parameter label: https://www.swift.org/documentation/docc/displayname.